scanner/src/main/kotlin/FileList.kt

```diff
@@ -31,7 +31,7 @@ import org.ossreviewtoolkit.utils.common.getDuplicates
 /**
  * The model to store a file list for a resolved provenance.
  */
-internal data class FileList(
+data class FileList(
```

Comment:
Why should this become public?

Reply:
I intended to ask about alll `internal` -> `public` modifications in this change.

Reply:
After thinking over it, I currently have no need for re-use of the logic to reduce the FileLists. I will change it back. This will reduce the number of changes.

Reply:
> I intended to ask about alll `internal` -> `public` modifications in this change.

Yes, that's exactly what my answer was referring to. It cannot stay `internal` if it's now used as a paramater type of a `public` function.

Reply:
This change was rolled back.